Makefile

```diff
@@ -1,3 +1,16 @@
+build:
+	sudo chmod 777 -R .db-data/
+	docker build -t bancho:latest .
+
+run:
+	docker-compose up bancho mysql redis
```

Comment:
Hindsight is 20/20

Is there any reason we _want_ to specify these explicitly?
Has caused issues for users who wish to remove/add containers to their compose